server/models/capture-feature.js

```diff
@@ -35,4 +36,10 @@ const createCaptureFeature = (captureFeatureRepo) => (async captureFeature => {
     repository.add(captureFeature);
 });
 
-module.exports = { captureFeatureFromMessage, createCaptureFeature };
+const updateCaptureFeature = (captureFeatureRepo) => (async (captureFeatureIds, captureFeatureUpdateObject) => {
+  log.warn("repo:", captureFeatureRepo);
+  // Because here is using a special fn to update db, so didn't use Repository
+  await captureFeatureRepo.batchUpdate(captureFeatureIds, captureFeatureUpdateObject);
+});

```

Comment:
Casually, I don't know why we need to use `new Repository` there.

Reply:
The idea was to give  models package with functions to have only local dependencies and explicitly stating what signatures are expected when passing an repository implementation. I agree its need is arguable.